test/Microsoft.DotNet.CommandFactory.Tests/GivenAProjectToolsCommandResolver.cs

```diff
@@ -355,34 +329,16 @@ public void ItFindsToolsLocatedInTheNuGetFallbackFolder()
                 .Should()
                 .Pass();
 
-            var commandResolverArguments = new CommandResolverArguments()
-            {
-                CommandName = "dotnet-fallbackfoldertool",
-                CommandArguments = null,
-                ProjectDirectory = testProjectDirectory
-            };
-
-            var result = projectToolsCommandResolver.Resolve(commandResolverArguments);
-
-            result.Should().NotBeNull();
-
-            var commandPath = result.Args.Trim('"');
-            commandPath.Should().Contain(Path.Combine(
-                fallbackFolder,
-                "dotnet-fallbackfoldertool",
-                "1.0.0",
-                "lib",
-                "netcoreapp3.0",
-                "dotnet-fallbackfoldertool.dll"));
+            new DotnetCommand()
```

Comment:
Change to end to end style since it is using stage0 SDK while the inserted SDK is not used